### PR TITLE
Script for install dependency of Ubuntu 18.04

### PR DESCRIPTION
**Pull request by plChao**
Programmer can install dependency for ubuntu by command `./install.sh ubuntu_1804`.
Remember to run `sudo ./install.sh everything` to install others pkg.

### CODE REVIEW

File: contrib/depend/install.sh [hunk 1]
```diff
@@ -126,6 +126,18 @@ xtensor_python() {
 
 }
 
+ubuntu_1804(){
+
```
Comment (yungyuc) [inline contrib/depend/install.sh]: There should not be a blank line.

File: contrib/depend/install.sh [hunk 2]
```diff
@@ -126,6 +126,18 @@ xtensor_python() {
...
+  sudo apt-get -qy install \
```
Comment (yungyuc) [inline contrib/depend/install.sh]: Your script should not invoke sudo.  Users should be allowed to opt in sudo when invoking the script from outside, like `sudo your_script.sh`.

File: contrib/depend/install.sh [hunk 3]
```diff
@@ -139,4 +151,6 @@ elif [ $1 == "xtensor-python" ]; then
 elif [ $1 == "everything" ]; then
   pybind11
   modmesh
+elif [ $1 == "ubuntu_1804" ]; then
```
Comment (yungyuc) [inline contrib/depend/install.sh]: It is a bad idea to mix installation scripts with a package manager and with source building.

**Comment by plChao** [comment]
How can I make changes to this pull request ?
Can I just make new commit to the fork of this repo and it will update to here?

**Comment by yungyuc** [comment]
Yes, adding new changes to the branch and pushing to your fork will update the PR.

**Comment by yungyuc** [comment]
Close for #17

**Pull request closed without merging**